static/js/components/tiles/chart_footer.tsx

```diff
@@ -29,6 +29,9 @@ interface ChartFooterPropType {
 }
 
 export function ChartFooter(props: ChartFooterPropType): JSX.Element {
+  if (!props.handleEmbed && !props.exploreLink) {
```

Comment:
did you mean for this to make it in?

Reply:
oh, yes, that was a small bug I found, let me add a note in the PR comment